### DISCUSSION TITLE
Class .selected applies to non-selected node

### DISCUSSION THREAD

**Discussion by insssomniac** [bcakmakoglu/vue-flow#933] [Q&A] [answered]
Hi there,

I'm not sure if it is a bug - when you drag&drop the node, the class .selected adds to it, but in fact it is not become selected and you can not get it with the 'getSelectedNodes' getter.

Practically node becomes selected only if you click on it, but not after drag&dropping. And it seems correct, so it would be better to apply class .selected in the same cases, isn't it?

**Comment by bcakmakoglu** [comment, marked as answer]
I assume by drag & drop you mean just moving a node around?
By default nodes *will be selected on drag*.
The class is therefore applied and `getSelectedNodes` would return that node (which it does, I've just tested it myself).

You can disable this behavior by setting `selectNodesOnDrag` to `false`.

**Comment by insssomniac** [comment]
Thanks! That's what I need
